exposed-core/build.gradle.kts

```diff
@@ -18,7 +18,7 @@ dependencies {
 }
 
 group = "in.porter.exposed"
-version = "0.21.1"
+version = "0.21.2"
```

Comment:
This was by mistake. It has been corrected to "0.21.1.1" in another [commit](https://github.com/porterin/Exposed/commit/1643f46f8a95c42a735718501ba27aed4bd36714).